test/integration/model/upsert.test.js

```diff
@@ -63,6 +63,8 @@ describe(Support.getTestDialectTeaser('Model'), () => {
         const [, created0] = await this.User.upsert({ id: 42, username: 'john' });
         if (['sqlite', 'postgres'].includes(dialect)) {
           expect(created0).to.be.null;
+        } else if (dialect === 'db2') {
+          expect(created0).to.be.undefined;
```

Comment:
I was close to point out that this is really unfortunate... however, PG doesn't seem to support it either 🤔